Documentation/SpatialAwareness/SpatialAwarenessGettingStarted.md

```diff
@@ -82,11 +83,14 @@ This feature enables rapid development and testing without requiring the build a
 to do final acceptance testing using a built and deployed version of the application, running on the target 
 hardware and platform.
 
+### Hiding the spatial mesh
+
+In order to hide the spatial mesh (assuming the default profile is in use, or the steps in [Enable Spatial Awareness](#enable-spatial-awareness)
```

Comment:
This is not quite accurate. Hiding the mesh is different from disabling the system.  Either the caption needs to be Disabling Spatial Awareness or the instructions need to direct the user to set the DisplayOption on the observer to None

Reply:
Removed this, added a code snippet to the usage guide.